### PR TITLE
multi-geojson file support & utils to find new files 

### PR DESCRIPTION
**Pull request by pramod-thupaki**

### CODE REVIEW

**Comment by n-a-t-e** [comment]
what is the 'find new files' stuff for?

**Comment by pramod-thupaki** [comment]
Its the code to find any new files that need to be converted to netcdf. Not applicable to ODF transform project right now; but useful for the IOS datasets ...

File: cioos_data_transform/cioos_data_transform/utils/compare_files.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import hashlib
+import os
+import json
+import glob
+
+
+def get_file_stats(flist):
+    info = {}
+    for f in flist:
+        info.update({f: [os.stat(f).st_size, int(os.stat(f).st_mtime / 60)]})
```
Comment (n-a-t-e) [inline cioos_data_transform/cioos_data_transform/utils/compare_files.py:10]: any reason for update() here? Could do `info[f]=[os.stat(f).st_size, int(os.stat(f).st_mtime / 60)]`

Comment (pramod-thupaki) [reply cioos_data_transform/cioos_data_transform/utils/compare_files.py]: good point, update() is not needed here...

File: cioos_data_transform/cioos_data_transform/utils/compare_files.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+def compare_files(file1, file2, method="cmp"):
+    if method == "cmp":
+        return os.stat(file1) == os.stat(file2)
+    elif method == "md5":
+        return hash_file(file1) == hash_file(file2)
+
+
+if __name__ == "__main__":
+    f = "/home/pramod/Downloads/ubuntu-20.04.2.0-desktop-amd64.iso"
```
Comment (n-a-t-e) [inline cioos_data_transform/cioos_data_transform/utils/compare_files.py:70]: is this for anything?

Comment (pramod-thupaki) [reply cioos_data_transform/cioos_data_transform/utils/compare_files.py:70]: its a large binary file to test which method for comparison is faster ... part of code for testing the module

**Pull request merged**